ApplicationCode/GrpcInterface/Python/rips/grid.py

```diff
@@ -32,3 +32,48 @@ def dimensions(self):
         return self.__stub.GetDimensions(
             Grid_pb2.GridRequest(case_request=case_request,
                                  grid_index=self.index)).dimensions
+
+    def cell_centers(self):
+        """The cell center for all cells in given grid
+
+        Returns:
+            Vec3d: class with double attributes x, y, x giving cell centers
+        """
+        case_request = Case_pb2.CaseRequest(id=self.case.case_id)
+        def get_cell_centers_async():
+            chunks = self.__stub.GetCellCenters(
+                Grid_pb2.GridRequest(case_request=case_request,
+                                 grid_index=self.index))
+
+            for chunk in chunks:
+                yield chunk
+
+        centers = []
+        chunks = get_cell_centers_async()
+        for chunk in chunks:
+            for center in chunk.centers:
+                centers.append(center)
+        return centers
+
+
+    def cell_corners(self):
+        """The cell corners for all cells in given grid
+
+        Returns:
+            list of CellCorners: a class with Vec3d for each corner (c0, c1.., c7)
+        """
+        case_request = Case_pb2.CaseRequest(id=self.case.case_id)
+        def get_cell_corners_async():
```

Comment:
move get_cell_corners_async outside and remove get_

Reply:
Fixed.